dist/images/Dockerfile.fedora

```diff
@@ -22,7 +22,8 @@ ARG TARGETOS
 ARG TARGETARCH
 
 WORKDIR /workspace
-RUN apt-get update && apt-get install -y -qq make git
+RUN dnf install --best --refresh -y make git && \
+    dnf clean all
```

Comment:
This is a build stage you don't need to clean it up.

Reply:
Done